Add workflow commenting release PRs with tests list (#3304)
tahowallet/extension: .github/workflows/test-list/release-test-list.md

# Release checklist

This release checklist should be performed before release is published.

- dev tests should be performed by the dev team
- if something is not working please create an issue and link it here. Checkbox should be checked only if everything was fine
- in case of serious issues or doubts you should ask the team first

**Don't forget to approve the PR when checklist is finished ✅**

---

## Dev tests

### 🚀 Version update

- [ ] update from previous version with some accounts loaded

---

## Manual tests

### 📨 Add account

1. Add read-only account with ENS
   - [ ] check assets
   - [ ] check balance
   - [ ] check NFTs
   - [ ] check abilities
   - [ ] check activities
   - [ ] check overview page
2. Add read-only account with UNS
   - [ ] check assets
   - [ ] check balance
   - [ ] check NFTs

>    - [ ] check abilities

Reviewer: I've only noticed it now. Abilities are not available to read-only accounts. It's a bit confusing and I think it's worth adding more precise information about what we want to check here.

Reviewer 2: @kkosiorowska ah sorry my bad, could you remove that and make a PR? Feel free to suggest other tweaks to the checklist. I still want it to be as short and concise as possible, we can assume our release checklist tester will know how to test these points. We can also do a Notion/Coda page with a description of how to test each point - not a need right now but in the future I guess that will be nice to have.

Reviewer: Sure, I will do it later today.

# Release checklist

This release checklist should be performed before release will be published.

- dev tests should be performed by the dev team
- if something is not working please create an issue and link it here. Checkbox should be checked only if everything was fine
- in case of serious issues or doubts you should ask the team first

> **Don't forget to approve the PR when checklist will be finished ✅**

Author: ```suggestion
**Don't forget to approve the PR when checklist is finished ✅**
```

<summary>📡 dApps connections</summary>

- [ ] connect to [Uniswap](https://app.uniswap.org/)
- [ ] connect to [Opensea](https://opensea.io/)
- [ ] connect to [Galxe](https://galxe.com/)
- [ ] connect to [PancakeSwap](https://pancakeswap.finance/)
- [ ] connect to [TraderJoe](https://traderjoexyz.com/avalanche)
- [ ] connect to [SpookySwap](https://spooky.fi/#/)
- [ ] connect to [Velodrome](https://app.velodrome.finance/swap)
- [ ] connect to [GMX](https://app.gmx.io/#/trade)
- [ ] connect to [Mintkudos](https://mintkudos.xyz/)
</details>

<details>
<summary>🎭 Sign in with Ethereum</summary>

- [ ] SIWE on [login.xyz](https://login.xyz/)
- [ ] SIWE on [Taho Pledge](https://taho.xyz/web3pledge)
</details>

<details>
<summary>🐶 Default wallet</summary>

1. Default wallet setting turned OFF
   - [ ] connect with dapp without MM available using Taho
   - [ ] connect with dapp with MM available using MM
2. Default wallet setting turnef ON
   - [ ] connect with dapp without MM available using Taho
   - [ ] connect with dapp with MM available using Taho
   </details>

<details>
<summary>⚙️ Settings</summary>

- [ ] check `hide balance under $2` option
- [ ] check bug reports - export logs

> </details>

Author: What do you think about adding some checks for other options that are not covered on the list?
For example:
```
- [ ] check enabling test networks - enable/disable, send funds on test network
- [ ] check connected websites - disconnect single / all
- [ ] check links - Need help, Discord, Twitter, GitHub
- [ ] check analytics - on/off, deleting data, privacy policy link
```

Reviewer 2: I would be careful with adding new things to this list - I would totally want much more things to be tested here but we want it test most important things.

> check enabling test networks

we had tests about test networks before on the checklist but as test networks are not that important to majority of users and they won't bring us revenue we decided to drop it

> check connected websites

this makes sense and is quick to test, will add

> check links 

links are changing very rarely, no need to test them often

> check analytics

mixed feelings - this is important but will be hard to manually test if data was actually deleted. But we should test it either with E2E or make the unit tests coverage better
